spec/features/user_can_pick_joke_spec.rb

```diff
@@ -6,16 +6,16 @@
     visit '/'
   end
   it "has a random joke" do
-    expect(page).to have_button('Pick a Joke for me!')
+    expect(page).to have_button('Pick a random joke for me!')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.